sapi.services.yml

```diff
@@ -1,7 +1,7 @@
 services:
   sapi.dispatcher:
     class: Drupal\sapi\Dispatcher
-    arguments: ["@config.manager", "@current_user", "@current_route_match", "@plugin.manager.sapi_action_handler", "@service_container"]
+    arguments: [ "@plugin.manager.sapi_action_handler" ]
```

Comment:
I should remove the spaces around @plugin.manage.sapi_action_handler in the sapi.dispatcher delcaration in sapi.services.yml